analyses/rna-seq-composition/rna-seq-composition.Rmd

```diff
@@ -0,0 +1,579 @@
+---
+title: "`r gsub('.Rmd', ' ', gsub('_', ' ', knitr::current_input()))`"
+author: "Holly Beale"
+date: "`r format(Sys.time(), '%a %b %d %X %Y')`"
+output:
+  html_document:
+    toc: yes
+
+---
+
+
+```{r}
+library(tidyverse) # present in image
+library(gridExtra) # present in image
+library(grid) # present in image
+
+library(EnvStats) # installed from MRAN in Dockerfile
+library(janitor) # installed from MRAN in Dockerfile
+library(patchwork) # installed from Github in Dockerfile
+
+```
+
+
+```{r}
+
+data_dir <- file.path("..", "..", "data")
+scratch_dir <- file.path("..", "..", "scratch")
```

Comment:
Since the unpacked data are not nested in a folder, it might be worth making a subdirectory of the `scratch` directory for ease of cleanup later.